src/router/Routes.tsx

```diff
@@ -59,16 +56,15 @@ const ContainedComponent = (props: ContainedComponentProps) => {
     )
 }
 
-export function signUp(){
-    return <SignUp/>
-}
-
 export function Home() {
+    const isAdmin = localStorage.getItem('admin');
+    let adminBool = false;
+    if(isAdmin === 'true') adminBool = true;
```

Comment:
You can avoid setting the `adminBool` variable, use the `isAdmin` variable directly. Doing this:
`const isAdmin = !!localStorage.getItem("admin");`